spack-ext/repos/spack-stack/packages/neptune-env/package.py

```diff
@@ -41,19 +41,18 @@ class NeptuneEnv(BundlePackage):
     depends_on("fftw", type="build")
     depends_on("netlib-lapack", type="build")
 
-    with when("+python"):
-        depends_on("py-f90nml", type="run")
-        depends_on("py-h5py", type="run")
-        depends_on("py-netcdf4", type="run")
-        depends_on("py-pandas", type="run")
-        depends_on("py-pycodestyle", type="run")
-        depends_on("py-pybind11", type="run")
-        depends_on("py-pyhdf", type="run")
-        depends_on("py-python-dateutil", type="run")
-        depends_on("py-pyyaml", type="run")
-        depends_on("py-scipy", type="run")
-        depends_on("py-xarray", type="run")
-        depends_on("py-xnrl", type="run")
-        depends_on("py-pytest", type="run")
+    depends_on("py-f90nml", type="run", when="+python")
```

Comment:
Does the 'with when' not work here?

Reply:
I think it does (it was a fool's attempt to fix a problem that I didn't understand). I can revert the change.

Reply:
Sounds good. Meanwhile the rest looks good so I'll go ahead and approve.